src/metrics/mod.rs

```diff
@@ -0,0 +1,46 @@
+//! Metrics collection
```

Comment:
we have switched to the new module layout, so this file should be named `src/metrics.rs`